### ISSUE TITLE
Turning thrusters don't show up on all ships

### ISSUE THREAD

**Issue by EatThePath** [scp-fs2open/fs2open.github.com#4395]
Originally from fightermedic on mantis http://scp.indiegames.us/mantis/view.php?id=3044

> thrusters do not show up on a ship that doesn't belong to the player's wing, but even then they don't most of the time furthermore, it appears "roll right" and "bank right" are switched in their functionallity, same goes for same goes for the left ones

The roll and bank are switched from standard FSO conventions yes, but fixing that now would really not be viable. More importantly, I can confirm the display issue, and it's rather limiting of some stuff WM would like to do in the future

**Comment by z64555** [comment]
This is assuming that the ships have thruster points to where the jets are to be drawn, yes?

**Comment by Baezon** [comment]
Yes, ETP works extensively with ships covered in maneuvering thrusters. The core issue is that the AI doesn't really 'export' it's rotational acceleration in any way, and unlike players do not use `control_info` for turning, so any thrusters associated with rotation do not function. Lateral thrusters work correctly, and using maneuvers causes them to work properly (because that switches to ´control_info´)
